Updated image dimensions

katikaa opened the pull request:
Resolves #485

Updating image dimensions, locators and origin codes for the `rawImage` block.

- [ ] ~Tests added for new features~
- [ ] Test engineer approval

**Index table**

- portrait image (9:16): scenario 2, 6 (scenario 6 contains an image that has the extension -nc.png which means it's currently not compressed by image chef)
- landscape image (16:9): scenario 3, 4, 5, 8, 11
- landscape images (not 16:9): scenario 12, 16
- very small image: scenario 15
- square image: scenario 22

katikaa committed:
Updated image dimensions

katikaa committed:
Updated text

katikaa committed:
Updated image dimensions

katikaa committed:
Update scenario-05.json

katikaa committed:
Updated image dimension

katikaa committed:
Updated image dimensions (nc)

katikaa committed:
Updated image dimensions

katikaa committed:
Updated images' dimensions

katikaa committed:
Updated images' dimensions

katikaa committed:
Updated image dimensions

sareh commented on data/scenario-02.json:
```diff
@@ -66,8 +66,8 @@
                 "model": {
-                  "width": 640,
-                  "height": 420,
+                  "width": 572,
```
```
"width": 2362,
"height": 2952,
```

katikaa replied:
This is a Visual Journalism image that has the extension -nc.jpg. That means that it's not compressed by image chef currently so as to retain its quality. i.e as sen in this story: https://www.bbc.co.uk/news/uk-43826163. Therefore I left its dimensions as we would receive them.

sareh commented on data/scenario-03.json:
```diff
@@ -139,7 +139,7 @@
                       "model": {
                         "width": 640,
-                        "height": 420,
+                        "height": 360,
```
```
"width": 1024,
"height": 576,
```

katikaa replied:
I tried to add images in various sizes so as to cover different users' devices. Let me know if you'd prefer smaller images :)

sareh commented on data/scenario-05.json:
```diff
@@ -104,7 +104,7 @@
                 "model": {
                   "width": 640,
-                  "height": 420,
+                  "height": 360,
```
```
"width": 1024,
"width": 576,
```

sareh commented on data/scenario-05.json:
```diff
@@ -232,7 +232,7 @@
                       "model": {
                         "width": 640,
-                        "height": 420,
+                        "height": 360,
```
```
"width": 1024,
"width": 576,
```

sareh commented on data/scenario-08.json:
```diff
@@ -153,7 +153,7 @@
                 "model": {
                   "width": 640,
-                  "height": 420,
+                  "height": 360,
```
```
"width": 1024,
"width": 576,
```

sareh commented on data/scenario-08.json:
```diff
@@ -264,7 +264,7 @@
                 "model": {
                   "width": 640,
-                  "height": 420,
+                  "height": 360,
```
```
"width": 1024,
"width": 576,
```

sareh commented on data/scenario-11.json:
```diff
@@ -193,7 +193,7 @@
                 "model": {
                   "width": 640,
-                  "height": 420,
+                  "height": 360,
```
```
"width": 1024,
"width": 576,
```

sareh commented on data/scenario-03.json:
```diff
@@ -57,7 +57,7 @@
                       "blockId": "1bf4d5c7-a386-32c5-4ad5-a037ead08760",
                       "model": {
-                        "text": "Lorem ipsum dolor sit amet, consectetur adipiscing elit ©"
+                        "text": "Lorem ipsum dolor sit amet, consectetur adipiscing elitelit ©"
```
Did you mean to make the text change here?

katikaa replied:
you can reject this one. I did in my 'github learning curve' while trying to understand if I can make changes in different files under the same PR :)

sareh reviewed (commented):
Thanks for making these changes! I noticed that a few of these had larger possible height/width values.

katikaa committed:
Updated images' dimensions

BogdanDogaru committed:
Merge branch 'latest' into katikaa-patch-1

Commit:
Update originCode data type in the rawImage schema

Commit:
Merge branch 'latest' of github.com:bbc/simorgh into katikaa-patch-1

katikaa committed:
Updated images' dimensions

katikaa committed:
Updated image dimensions

katikaa committed:
Updated images' dimensions

katikaa committed:
Updated images' dimensions

katikaa committed:
Updated images' dimensions

Some of the image urls (image locator) are of this format: https://ichef.bbci.co.uk/images/ic/976x549/p06704bb.jpg. I don't know why that was, I have left that unchanged

sareh commented on data/prod/scenario-25.json:
```diff
@@ -142,8 +142,8 @@
                 "model": {
                   "width": 640,
                   "height": 420,
```
The image ratio here is not 16:9, whereas the image itself is 16:9. 
As soon as we use the height value as well as the width value in our code, this mismatch between the ratio in the data and that of the original image will make images on our Test/Live scenario pages look skewed. 
Because of this, could you please update all the landscape images to have a 16:9 ratio in the data, such as :
```
"width": 1024,
"height": 576,
```

Commit:
Revert "Update originCode data type in the rawImage schema"

This reverts commit 3ee40853feafe622fc460dbab4c60fa800a24d6d.

katikaa committed:
Updated images' dimensions

katikaa committed:
Update image's dimensions

katikaa committed:
Updated image's dimensions

katikaa committed:
Updated images' dimensions

katikaa committed:
Updated images' dimensions

katikaa committed:
Updated images' dimensions

sareh committed:
Merge branch 'latest' into katikaa-patch-1

sareh committed:
Merge branch 'latest' into katikaa-patch-1

BogdanDogaru committed:
Merge branch 'latest' into katikaa-patch-1

sareh committed:
Merge branch 'latest' into katikaa-patch-1

sareh committed:
Update image dimensions

sareh committed:
Merge branch 'latest' into katikaa-patch-1

sareh committed:
Undo text change
